components/omega/src/ocn/Tendencies.cpp

```diff
@@ -398,6 +426,27 @@ void Tendencies::computeVelocityTendenciesOnly(
           });
    }
 
+   // Compute wind forcing
+   const auto &NormalStressEdge = AuxState->WindForcingAux.NormalStressEdge;
+   const auto &MeanLayerThickEdge =
+       AuxState->LayerThicknessAux.MeanLayerThickEdge;
+   if (LocWindForcing.Enabled) {
+      parallelFor(
+          {NEdgesAll, NChunks}, KOKKOS_LAMBDA(int IEdge, int KChunk) {
+             LocWindForcing(LocNormalVelocityTend, IEdge, KChunk,
+                            NormalStressEdge, MeanLayerThickEdge);
+          });
+   }
+
+   // Compute bottom drag
+   if (LocBottomDrag.Enabled) {
+      parallelFor(
+          {NEdgesAll, NChunks}, KOKKOS_LAMBDA(int IEdge, int KChunk) {
```

Comment:
not sure why we need a loop over NChunks.  I guess if Omega-0 is planning to do wind with multiple layers that's fine, but we'll have to change it later.

Reply:
I changed the bottom drag term to loop only over the edges in the bottom layer. For the wind stress term I kept the loop over multiple layers, since that's what is done in MPAS-O with some coefficient to distribute the stress that depends on the vertical coordinate. Once we have a vertical coordinate we can do the same in Omega.